packages/utilities/fast-colors/src/color-blending.ts

```diff
@@ -316,7 +320,7 @@ export function blend(
         case ColorBlendMode.Burn:
             return blendBurn(bottom, top);
         case ColorBlendMode.Color:
-            return blendColor(bottom, top);
+            return blendColor(bottom, top)!;
```

Comment:
I don't think trailing exclamation point shouldn't be necessary if we're handling null as expected.

Reply:
Yes, I can remove that too. Removing the trailing exclamation point will lead the `blend` function to return `ColorRGBA64 | null`.